index.js

```diff
@@ -835,7 +1441,7 @@
         }
         var $typeVarMap = {};
         for (var typeVarName in _typeVarMap) {
-          $typeVarMap[typeVarName] = _typeVarMap[typeVarName].slice();
+          $typeVarMap[typeVarName] = _typeVarMap[typeVarName];
```

Comment:
@kedashoe and I discussed this on Gitter and agreed that cloning the arrays has never been necessary. My guess is that while working on the initial version I used `R.clone`, and later, when I removed the Ramda dependency, wrote this code to perform a "deep" clone manually.